src/character.cpp

```diff
@@ -11631,53 +11632,76 @@ void Character::pause()
         try_remove_grab();
     }
 
-    // put pressure on bleeding wound, prioritizing most severe bleeding
+    // put pressure on bleeding wound, prioritizing most severe bleeding that you can compress
     if( !is_armed() && has_effect( effect_bleed ) ) {
+        // Calculate max staunchable bleed level
+        // Top out at 20 intensity for base, unencumbered survivors
+        int max = 20;
+        max *= get_modifier( character_modifier_bleed_staunch_mod );
+        add_msg_debug( debugmode::DF_CHARACTER, "Staunch limit after limb score modifier %d", max );
+
+        // +5 bonus if you know your first aid
+        if( has_proficiency( proficiency_prof_wound_care ) ||
+            has_proficiency( proficiency_prof_wound_care_expert ) ) {
+            max += 5;
+            add_msg_debug( debugmode::DF_CHARACTER, "Wound care proficiency found, new limit %d", max );
+        }
+
+        int num_broken_arms = get_num_broken_body_parts_of_type( body_part_type::type::arm );
+        int num_arms = get_num_body_parts_of_type( body_part_type::type::arm );
+
+        // Don't warn about encumbrance if your arms are broken
+        if( num_broken_arms ) {
+            // Handle multiple arms
+            max *= ( 1.0f - num_broken_arms / num_arms );
```

Comment:
This will do integer division, you'll get either 1 or 0.
```suggestion
            max *= ( 1.0f - num_broken_arms / static_cast<float>( num_arms ) );
```